modules/apps/static/portal-osgi-web/portal-osgi-web-servlet-context-helper-impl/src/main/java/com/liferay/portal/osgi/web/servlet/context/helper/internal/CustomServletContextHelper.java

```diff
@@ -95,6 +96,11 @@ public void contextInitialized(ServletContextEvent servletContextEvent) {
 			bundleWiring.getClassLoader());
 	}
 
+	@Override
+	public String getMimeType(String name) {
+		return MimeTypesUtil.getContentType(name);
```

Comment:
Could use _servletContext.getMimeType(name) instead?

Reply:
@tototrinh 
It seems that it would cause the recursive call and StackOverFlow.